components/threading/global_threads/thread_pane/thread_pane.test.tsx

```diff
@@ -13,6 +13,9 @@ import Header from 'components/widgets/header';
 import FollowButton from 'components/threading/common/follow_button';
 import Button from 'components/threading/common/button';
 
+import TestHelper from '../../../../packages/mattermost-redux/test/test_helper';
+import {UserProfile} from '@mattermost/types/users';
```

Comment:
minor nit : import rearrangments

Reply:
Is it really important, why don’t we add this in eslint.